### ISSUE TITLE
Unused import is not reported, when there is companion object with the same name

### ISSUE THREAD

**Issue by YuraLaguta** [pinterest/ktlint#563]
In AS:
reported:
![Screen Shot 2019-08-23 at 11 50 37 am](https://user-images.githubusercontent.com/1853043/63561206-6b765d00-c59c-11e9-9229-30b1297492cd.png)
Usage like: `Log.e(TAG, "bla bla")`
companion object takes precedence
```
   companion object {
        const val TAG = "MyTag"
    }
```

**Comment by paul-dingemans** [comment]
This issue can not be reproduced. I have tried with lots of olds version (back until 0.34.0) as well.
